rocks-bin/src/main.rs

```diff
@@ -1,13 +1,26 @@
 use std::{path::PathBuf, time::Duration};
 
 use clap::{Parser, Subcommand};
-use rocks_lib::config::Config;
+use rocks_lib::config::{Config, LuaVersion};
 
+mod build;
 mod download;
 mod rockspec;
 mod search;
 mod unpack;
 
+fn parse_lua_version(s: &str) -> Result<LuaVersion, String> {
+    match s {
+        "5.1" | "51" | "jit" | "luajit" => Ok(LuaVersion::Lua51),
```

Comment:
question: would `luajit` be considered valid? I recall seeing that somewhere.

Reply:
LuaJIT can be a valid target (since mlua can be compiled to use it), so sure, this should be allowed. Luarocks itself doesn't recognize this though.

What I'm curious about is whether it could be possible to supply custom parameters here based on a list defined in e.g. a config file. That could allow us to use Neovim as an interpreter super easily!